tests/unit/test_utils.py

```diff
@@ -2,11 +2,11 @@
 import pandas as pd
 import pytest
 
-from fold.composites.common import traverse, traverse_apply
 from fold.composites.target import TransformTarget
 from fold.transformations.dev import Lookahead, Test
 from fold.transformations.difference import Difference
 from fold.transformations.math import TakeLog
+from fold.traverse import traverse, traverse_apply
```

Comment:
Moved traverse functions to the root in a file named `traverse.py`

Reply:
I'm a bit unsure about this, as that'll be directly accessible to the public, maybe I'd just move it to `base.py` singifying that it's not for public consumption, or prefix the function names with `_` (started doing it in a couple of places)

Reply:
don't worry about it for now